usr/src/uts/armv8/os/ddi_impl.c

```diff
@@ -2277,10 +2279,85 @@ get_boot_properties(void)
 	kmem_free(bop_staging_area, MMU_PAGESIZE);
 }
 
+static void
```

Comment:
I'm very confused by this.  This _should_ be working.  For a start, QEMU,virt has an "interrupt-parent" on /, and nothing will work if that doesn't happen?

These should trickle up from the PROM layer through the DDI interfaces via `impl_ddi_bus_prop_op`

Reply:
Wait, I didn't intend to approve, I intended to comment on this, and apparently I missed.

Reply:
> I'm very confused by this. This _should_ be working. For a start, QEMU,virt has an "interrupt-parent" on /, and nothing will work if that doesn't happen?
> 
> These should trickle up from the PROM layer through the DDI interfaces via `impl_ddi_bus_prop_op`

Yeah, it should - I have no idea why they don't show in `prtconf` as hardware properties. A fair bit of the ddi_impl code still uses raw `prom_*` interfaces for property lookup, which I'm looking to clean up, since this is really device tree lookup, rather than prom-specific (get_address_cells, get_size_cells, get_dma_ranges, getlongprop_buf, check_status and status_okay).